### ISSUE TITLE
Move payments page on MyProudCity to 'Payments' subnav

### ISSUE THREAD

**Issue by lukefretwell** [proudcity/wp-proudcity#2484]
**Source**
The source of the issue (ex: Customer: #CustomerName#).

LF

**What feature would you like to improve**
A clear and concise description of the current feature.

MyProudCity clean-up.

**Describe the solution you'd like**

Per discussion, move payments dashboard to a subnav under 'Payments' in admin. Title this 'Payments reports' or 'Reports'.

**Additional context**

**Comment by kevindherman** [comment]
@curtismchale @lukefretwell correct me if I'm wrong but my understanding was that PC Payments would have a link in the admin menu as mentioned here but the management of PC Payments would be through the new my.proudcity.com settings. 

Unless this is more long term to migrate PC Payments out of my.proudcity.com altogether. That seems like a big lift for not a lot of use cases.

**Comment by curtismchale** [comment]
@kevindherman yes we can detect this on the WordPress end. When a user is given the `payments` permission in the PC Dashboard I can send over a capability to WP called `pc_payments` (or whatever) and then only show the menu to users that have this capability.

I made a note of this in the top of the issue and in my notes for the new PC Dashboard that aren't public yet.
